core.js

```diff
@@ -89,21 +81,23 @@ exports.init = function (sbot, config) {
     rimraf.sync(config.path)
     mkdirp.sync(config.path)
   }
+  const feedFormats = []
+  const encryptionFormats = []
   const indexes = {}
   const dir = config.path
   const privateIndex = PrivateIndex(dir, sbot, config)
-  const log = Log(dir, config, privateIndex)
+  const log = Log(dir, config, privateIndex, self)
   const jitdb = JITDB(log, jitIndexesPath(dir))
   const status = Status(log, jitdb)
   const debug = Debug('ssb:db2')
-  const post = Obv()
+  const post = Obv() // FIXME: move to compat?
```

Comment:
Yes pleased! This is used in ebt, but that will change with https://github.com/ssbc/ssb-ebt/pull/68 anyway. This seems to be only used in migrate inside db2. Can you check manyverse if there are other places?

Reply:
In Manyverse, `post` is only used in `ssb-ebt` and `ssb-conn/scheduler`. Can easily be replaced